Sources/AppKitBackend/AppKitBackend.swift

```diff
@@ -641,6 +641,22 @@ public final class AppKitBackend: AppBackend {
             onChange(textField.stringValue)
         }
         textField.onSubmit = onSubmit
+
+        if #available(macOS 14, *) {
+            textField.contentType =
+                switch environment.textContentType {
+                    case .url:
+                        .URL
+                    case .phoneNumber:
+                        .telephoneNumber
+                    case .name:
+                        .name
+                    case .emailAddress:
+                        .emailAddress
+                    default:
```

Comment:
Switch over remaining cases so that we're forced to evaluate how to handle new text content types when we add them in future (same goes for other backend implementations).

Reply:
> same goes for other backend implementations

I'm updating AppKit now, but GTK doesn't use textContentType at all, and UIKit and WinUI don't have `default:` branches. Could you clarify what you meant by this?

Reply:
At the time I left that comment I hadn't read through the other backends yet, so I just added that as a catch all. Only happens to apply to AppKitBackend though, so ignore that